.github/workflows/daily-e2e-tests-ibmcloud.yaml

```diff
@@ -4,15 +4,13 @@
 # Build and push container images for each cloud provider.
 ---
 name: daily e2e tests for ibmcloud
+
 on:
   schedule:
     # Runs "at 05:00(UTC time) every day" (see https://crontab.guru)
     # will base on default branch `main`
     - cron: '0 5 * * *'
-  push:
-    branches:    
-      - daily-e2e-ibmcloud
-      - develop
```

Comment:
@katexochen a newbie question: will workflow_dispatch work for any branch in local fork ?

Reply:
Exactly. You can use `gh workflow run --ref <branch-name>` or select the branch in the web UI:
![image](https://github.com/confidential-containers/cloud-api-adaptor/assets/49727155/705473b0-9d4e-4f29-b712-329dc48a1ae3)